packages/css/src/viewTransition.ts

```diff
@@ -0,0 +1,7 @@
+import { generateIdentifier } from './identifier';
+
+// createViewTransition is used for locally scoping CSS view transitions
+export const createViewTransition = (name: string, debugId?: string) => {
```

Comment:
We generally rely on debug styles for local dev, so there's not really much benefit to adding a named suffix to the identifier. It will also stick around in production styles, which isn't ideal. IMO this should basically be identical to the `createContainer` function.

Reply:
My thinking here was that you might want to specifically name a view transition, but it makes sense that you would just use debugId for that. I'll change it!